updating/index.adoc

```diff
@@ -0,0 +1,67 @@
+:_content-type: ASSEMBLY
+[id="updating-clusters-overview"]
+= Updating clusters overview
+include::_attributes/common-attributes.adoc[]
+:context: updating-clusters-overview
+
+toc::[]
+
+You can update an {product-title} 4 cluster with a single operation by using the web console or the OpenShift CLI (oc).
+
+[id="updating-clusters-overview-understanding-container-platform-service"]
+== Understanding the OpenShift Update Service
+xref:../updating/understanding-the-update-service.adoc#understanding-the-update-service[Understanding the OpenShift Update Service]: For clusters with internet accessibility, Red{nbsp}Hat provides over-the-air updates through an OpenShift Update Service as a hosted service located behind public APIs.
+
+[id="updating-clusters-overview-installing-and-configuring-service"]
+== Installing and configuring the OpenShift Update Service
+xref:../updating/installing-update-service.adoc#installing-update-service[Installing and configuring the OpenShift Update Service]: For clusters with internet accessibility, Red{nbsp}Hat provides over-the-air updates through an {product-title} update service as a hosted service located behind public APIs. If you are on a restricted network where disconnected clusters cannot access the public APIs, you can install the OpenShift Update Service locally. For more information, see the following procedures:
```

Comment:
You could remove the `{nbsp}` from this line, unless there is a specific reason for it.

Reply:
Ack'ed